internal/confmapprovider/discovery/bundle/bundle.d/receivers/redis.discovery.yaml.tmpl

```diff
@@ -0,0 +1,71 @@
+{{ receiver "redis" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)redis"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)redis" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)redis"
+  config:
+    default:
+      endpoint: {{ defaultValue }}
+      password: {{ defaultValue }}
+      username: {{ defaultValue }}
```

Comment:
I'm not too familiar with the redis acl system but I think default installations don't require auth info. Does it make sense not to use defaults for the username and password fields? They are still configurable by discovery config properties so I think the guidance can remain as is but we'd not be set up to fail to connect in all cases.

Reply:
This is a good call out! Will change that so we don't use defaults but leave the statement guidance